### PR TITLE
[PREVIEW] RDM-2224 - Resume Draft

### PR DESCRIPTION
**Pull request by tzarsmango**
https://tools.hmcts.net/jira/browse/RDM-2224

This one is for the user to resume editing the `CreateCase` event. Works with the [RDM-2224](https://github.com/hmcts/ccd-data-store-api/tree/RDM-2224) branch of the [case-data-store-api](https://github.com/hmcts/ccd-data-store-api)

Contains the changes in this PR, hence the big number of changed  files

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/shared/event-log/event-log-table.component.spec.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ describe('EventLogTableComponent', () => {
     let firstEvent = EVENTS[0];
 
     expect(firstRowCells[COL_EVENT].nativeElement.textContent).toBe(firstEvent.event_name);
-    expect(firstRowCells[COL_DATE].nativeElement.textContent).toBe('10 May 2017, 10:00:00 AM');
+    expect(firstRowCells[COL_DATE].nativeElement.textContent).toBe('10 May 2017, 10:00:00 AM UTC: 10 May 2017, 10:00:00 AM');
```
Comment (andrewfolga) [inline src/app/shared/event-log/event-log-table.component.spec.ts]: Why are we changing this?

File: src/app/shared/event-log/event-log-table.component.spec.ts [hunk 2]
```diff
@@ -102,7 +102,7 @@ describe('EventLogTableComponent', () => {
     let secondEvent = EVENTS[1];
 
     expect(secondRowCells[COL_EVENT].nativeElement.textContent).toBe(secondEvent.event_name);
-    expect(secondRowCells[COL_DATE].nativeElement.textContent).toBe('9 May 2017, 4:07:03 PM');
+    expect(secondRowCells[COL_DATE].nativeElement.textContent).toBe('9 May 2017, 4:07:03 PM UTC: 9 May 2017, 4:07:03 PM');
```
Comment (andrewfolga) [inline src/app/shared/event-log/event-log-table.component.spec.ts]: Why are we changing this?

File: src/app/shared/event-log/event-log-table.html [hunk 3]
```diff
@@ -12,7 +12,9 @@
            [target]="'_blank'"
            *ngIf="event.state_id !== 'Draft'">{{event.event_name}}</a><span
       *ngIf="event.state_id === 'Draft'">{{event.event_name}}</span></td>
-    <td>{{event.timestamp | ccdDate: 'short'}}</td>
+    <td><div class="tooltip">{{event.timestamp | ccdDate : 'local'}}
```
Comment (andrewfolga) [inline src/app/shared/event-log/event-log-table.html]: Why are we changing this?

File: src/public/config.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
   "pagination_page_size": 25,
   "postcode_lookup_url": "http://localhost:3453/addresses?postcode=${postcode}",
   "print_service_url": "http://localhost:3453/print",
-  "remote_document_management_url": "https://api-gateway.dev.dm.reform.hmcts.net/documents",
+  "remote_document_management_url": "http://localhost:4603/documents",
```
Comment (andrewfolga) [inline src/public/config.json]: Is this intentional?

Comment (tzarsmango) [reply src/public/config.json]: This is for working with local DM store instance, without this the document URLs are broken. This parameter is overridden in environments anyway...

File: src/app/shared/event-log/event-log-table.html (hunk 3 above)
Comment (tzarsmango) [reply src/app/shared/event-log/event-log-table.html]: @MKaramath added a local date modifier and a tooltip for the event details component, however this table seems to have been forgotten. Which leads different time displayed in log table and detail. This is just for consistency.

File: src/app/shared/event-log/event-log-table.component.spec.ts (hunk 2 above)
Comment (tzarsmango) [reply src/app/shared/event-log/event-log-table.component.spec.ts]: @MKaramath added a local date modifier and a tooltip for the event details component, however this table seems to have been forgotten. Which leads different time displayed in log table and detail. This is just for consistency.

File: src/app/shared/event-log/event-log-table.component.spec.ts (hunk 1 above)
Comment (tzarsmango) [reply src/app/shared/event-log/event-log-table.component.spec.ts]: @MKaramath added a local date modifier and a tooltip for the event details component, however this table seems to have been forgotten. Which leads different time displayed in log table and detail. This is just for consistency.

Comment (andrewfolga) [reply src/app/shared/event-log/event-log-table.component.spec.ts]: 👍

File: src/app/shared/event-log/event-log-table.component.spec.ts (hunk 2 above)
Comment (andrewfolga) [reply src/app/shared/event-log/event-log-table.component.spec.ts]: 👍

File: src/app/shared/event-log/event-log-table.html (hunk 3 above)
Comment (andrewfolga) [reply src/app/shared/event-log/event-log-table.html]: 👍

File: src/public/config.json (hunk 4 above)
Comment (andrewfolga) [reply src/public/config.json]: Ok fair enough.

**Pull request merged**